doc/nrf/config_and_build/bootloaders/index.rst

```diff
@@ -51,12 +51,14 @@ You can find an overview of currently supported bootloaders in the table below:
      - Yes
      - :doc:`See imgtool <mcuboot:imgtool>`
      - No
-     - Yes
+     - Yes `*`
      - Yes
      - :ref:`Monotonic (HW) <bootloader_monotonic_counter>`, :ref:`Semantic (SW) <ug_fw_update_image_versions_mcuboot>`
      - Image swap - single primary
        Dual-slot direct-xip
 
+`*` Only for Bluetooth® Low Energy and serial UART. (See the :ref:`ug_fw_update` page to learn about other DFU methods.)
```

Comment:
I suggest this * is handled like the * here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/zigbee.html